storm-server/src/main/java/org/apache/storm/scheduler/resource/ResourceAwareScheduler.java

```diff
@@ -113,6 +119,9 @@ public void schedule(Topologies topologies, Cluster cluster) {
             } else {
                 User submitter = userMap.get(td.getTopologySubmitter());
                 scheduleTopology(td, cluster, submitter, orderedTopologies);
+                if (!evictedTopologies.isEmpty()) {
+                    LOG.warn("Evicted Topologies {} when scheduling topology: {}", evictedTopologies, td.getId());
```

Comment:
evictedTopologies is cumulated during scheduling all the topologies since it only clears `evictedTopologies` at Line 111. So `evictedTopologies` here include all the evictedTopologies not only evicted by scheduling this topology `td.getId()`